### PR TITLE
Incremented version to 2.0-rc1.

### PR DESCRIPTION
**Pull request by dblock**
Signed-off-by: dblock <dblock@amazon.com>

### Description

Followup on https://github.com/opensearch-project/opensearch-build/pull/1863, incremented version to rc1.
  
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by gaiksaya — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -36,8 +36,8 @@ import org.opensearch.gradle.test.RestIntegTestTask
 buildscript {
     ext {
         isSnapshot = "true" == System.getProperty("build.snapshot", "true")
-        opensearch_version = System.getProperty("opensearch.version", "2.0.0-alpha1-SNAPSHOT")
-        buildVersionQualifier = System.getProperty("build.version_qualifier", "alpha1")
+        opensearch_version = System.getProperty("opensearch.version", "2.0.0-rc1-SNAPSHOT")
+        buildVersionQualifier = System.getProperty("build.version_qualifier", "rc1")
         // Taken from https://github.com/opensearch-project/alerting/blob/main/build.gradle#L33
         // 1.0.0 -> 1.0.0.0, and 1.0.0-SNAPSHOT -> 1.0.0.0-SNAPSHOT
```
Comment (peterzhuamazon) [inline build.gradle:42]: Can you remove these two lines and replace with what we have in other plugins?
Thanks.

Comment (dblock) [reply build.gradle:42]: Can we please not feature creep this PR and do this separately? :)

Comment (peterzhuamazon) [reply build.gradle:42]: You already change this part in other PRs.

Comment (dblock) [reply build.gradle:42]: ok

**Pull request closed without merging**